client/components/plans/plan-pill/style.scss

```diff
@@ -27,3 +27,13 @@
 	border-radius: 10px; /* stylelint-disable-line scales/radii */
 	background-color: var(--color-neutral-100);
 }
+
+.plan-pill.is-in-marketplace {
+	top: unset;
+	bottom: unset;
+	right: unset;
+	left: 50%;
+	transform: translateX(-50%);
```

Comment:
What's this doing?

Reply:
It aligns the pill to the center, please check [this comment](https://github.com/Automattic/wp-calypso/pull/67948#issuecomment-1251742427).